clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractStickyAssignorTest.java

```diff
@@ -640,15 +628,13 @@ public void testNoExceptionThrownWhenOnlySubscribedTopicDeleted() {
         partitionsPerTopic.put(topic, 3);
         subscriptions.put(consumerId, new Subscription(topics(topic)));
         Map<String, List<TopicPartition>> assignment = assignor.assign(partitionsPerTopic, subscriptions);
-        subscriptions.put(consumerId,
-            new Subscription(topics(topic), StickyAssignor.serializeTopicPartitionAssignment(
-                new ConsumerUserData(assignment.get(consumerId), Optional.of(1)))));
+        subscriptions.put(consumerId, buildSubscription(topics(topic), assignment.get(consumerId)));
 
         assignment = assignor.assign(Collections.emptyMap(), subscriptions);
         assertEquals(assignment.size(), 1);
         assertTrue(assignment.get(consumerId).isEmpty());
     }
-
+/*
```

Comment:
Is this intentional?

Reply:
I'm still cleaning up the tests 😄 (requires some refactoring, commented out ones that don't  yet compile)